cmd/zetaclientd/start.go

```diff
@@ -5,6 +5,9 @@ import (
 	"encoding/json"
 	"fmt"
 	"io"
+	"net/http"
+	// #nosec G108 -- pprof enablement is intentional
+	_ "net/http/pprof"
```

Comment:
Out of curiosity, why it is not used?

Reply:
The pprof handlers are registered via `init()` hook: https://cs.opensource.google/go/go/+/refs/tags/go1.23.3:src/net/http/pprof/pprof.go;l=95